azure-pipelines.yml

```diff
@@ -0,0 +1,29 @@
+trigger:
+  - master
+jobs:
+  - job: 'test_airflow'
+    pool:
+      vmImage: 'Ubuntu-16.04'
```

Comment:
given that our prod machines are ec2-linux, perhaps use RHEL is possible?

Reply:
Good point, I'll give that a shot.

Reply:
Looks like RHEL is not one of the [available VMs](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops#use-a-microsoft-hosted-agent)